src/runtime_src/core/tools/common/ReportCu.cpp

```diff
@@ -129,7 +129,14 @@ populate_cus(const xrt_core::device *device)
     }
   }
 
-  return pt;
+  boost::property_tree::ptree ptree;
+  try {
+    std::string uuid = xrt::uuid(xrt_core::device_query<xrt_core::query::xclbin_uuid>(device)).to_string();
+    boost::algorithm::to_upper(uuid);
+    ptree.put("xclbin_uuid", uuid);
+  } catch (...) {  }
+  ptree.add_child("compute_units", pt);

```

Comment:
Why is there another node called "compute_units" in the JSON tree.  This looks wrong.

Reply:
Thanks for the chat. I have made the changes that we dicscussed